src/table_builder_test.cc

```diff
@@ -429,6 +429,47 @@ TEST_F(TableBuilderTest, DictCompress) {
 #endif
 }
 
+TEST_F(TableBuilderTest, DictCompressOptions) {
```

Comment:
Please write the test in the Titan level, that is to say, new a titan instance and write some by db interface instead of putting into `TitanTableBuilder` directly